spec/unit/provider/oneview_hypervisor_cluster_profile_c7000_spec.rb

```diff
@@ -17,7 +17,7 @@
 require 'spec_helper'
 
 provider_class = Puppet::Type.type(:oneview_hypervisor_cluster_profile).provider(:c7000)
-api_version = 800
+api_version = 1800
```

Comment:
is there any reason for changing this api version here?

Reply:
Added UT for soft delete.

Reply:
add the same for synergy spec also

Reply:
Done